package.json

```diff
@@ -40,7 +40,7 @@
     "build:profile": "rm -rf build && cp envConfig/local.env .env && IS_PROD_PROFILE=true NODE_ENV=production webpack",
     "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
     "build:live:debug": "rm -rf build && awk '{sub(/LOG_DIR=.+/,\"LOG_DIR='log'\")}1' envConfig/live.env > .env && NODE_ENV=production webpack",
-    "build:preview": "cp envConfig/preview.env .env && NODE_ENV=production webpack",
+    "build:preview": "cp envConfig/preview.env .env && NODE_ENV=development webpack",
```

Comment:
@alex-magana I am hoping that this might be resolved when we upgrade to the latest major version of loadable 🤞 as I think there might be some ESM support: See #11073

Reply:
Thanks for the heads-up @karinathomasbbc.
I'll keep an eye out to check if the loadable upgrade fixes this issue
for us.